### ISSUE TITLE
Seems to download the first version of the CIA instead of latest

### ISSUE THREAD

**Issue by rlaphoenix** [tranxuanthang/villain3ds#16]
For example, Ultra Moon (W) is 0.0.0, Kirby Planet Robobot (W) is 0.1.0.
Is there any chance to get this fixed?

**Comment by tranxuanthang** [comment]
I'll investigate this later, right now I don't have my 3DS :(
If using an old version of CIA (eg: ultramoon) doesn't nintendo will require a full 3.5GB upgrade (redownload the newer CIA) to play that title?

**Comment by el-duderino5** [comment]
@PR4GM4 , isn't that how it's supposed to work though? The game gets released, and then an update is released on the eShop if it needs it? I know for a fact that UltraSun and UltraMoon have those updates out there in the eShop and I updated Ultrasun to the latest version with the update as soon as it was available.

**Comment by rlaphoenix** [comment]
Right, but sometimes the actual game game gets updated, for ex mario kart 7.

**Comment by el-duderino5** [comment]
Yeah, I know what you mean. It's the same case with monster hunter stories where the Japanese release got an update (1.1) before being released in other parts of the world, and you did not have to download that specific update since it was already integrated into the game.

Anyways, have you tried installing the updates for the games you mentioned and see if you run into any kind of problem? Also, where did you find information about the version number of those games?
